project/Dependencies.scala

```diff
@@ -26,7 +26,7 @@ object Dependencies {
   val workbenchOauth2V = s"0.5-$workbenchLibsHash"
   val workbenchAzureV = s"0.6-$workbenchLibsHash"
 
-  val helmScalaSdkV = "0.0.20"
+  val helmScalaSdkV = "0.0.8.5"
```

Comment:
This seems like a downgrade?

Reply:
I know, something fishy happened in helm scala sdk, but I triple checked on artifactory and this is the latest published version

Reply:
Hmm I wonder if I did something wonky with the version when I was updating it...